main.py

```diff
@@ -1,22 +1,37 @@
-from flask import Flask, request
-from flask_json import FlaskJSON
+import json
+from flask import Flask, request, make_response
 
 app = Flask(__name__)
-FlaskJSON(app)
 
-@app.route("/")
-def hello():
-    return "Hello World!"
+class ContentError(Exception):
+    pass
+class LogError(Exception):
+    pass
 
 @app.route('/login', methods=['POST'])
 def login():
+    try:
+        if request.headers['Content-Type'] == 'application/json':
+            data = json.loads(request.get_data())
+            if 'name' and 'password' in data.keys():
+                if data['name'] == 'denchik' and data['password'] == 'foobar':
+                    message = 'OK'
+                    http_code = 200
+                else:
+                    raise LogError
```

Comment:
Using exceptions for control flow is an antipattern, it's better to return from here.
https://softwareengineering.stackexchange.com/questions/189222/are-exceptions-as-control-flow-considered-a-serious-antipattern-if-so-why